tests/python/gaia-ui-tests/gaiatest/tests/unit/test_wifi_multiple_connect.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.marionette_test import parameterized
+
+from gaiatest import GaiaTestCase
+
+
+class TestWiFiMultipleConnect(GaiaTestCase):
+
+    @parameterized("1st_run")
+    @parameterized("2nd_run")
```

Comment:
I'll do that. I would like to know where to put this test in, as you proposed to be part of sanity testing. In which functional area should it be part of? 
It should not be part of unit testing, because it's really not a unit test.